src/AutoBuild.jl

```diff
@@ -1076,7 +1076,7 @@ function build_jll_package(src_name::String,
     mkpath(joinpath(code_dir, "src", "wrappers"))
 
     # Drop BuildDependency objects
-    dependencies = Dependency[d for d in dependencies if is_runtime_dependency(d)]
+    dependencies = filter(is_runtime_dependency, dependencies)
```

Comment:
The reason why I did the comprehension with `Dependency[...]` is because
```
julia> dep = [Dependency("Foo_jll"), BuildDependency("Bar_jll")]
2-element Vector{AbstractDependency}:
 Dependency(PackageSpec(
  name = Foo_jll
  version = *
), nothing)
 BuildDependency(PackageSpec(
  name = Bar_jll
  version = *
))

julia> filter(is_runtime_dependency, dep)
1-element Vector{AbstractDependency}:
 Dependency(PackageSpec(
  name = Foo_jll
  version = *
), nothing)
```
but I wanted the output to be `Vector{Dependency}` (there may be tests making sure of this)

Reply:
Fixed, oops